host/cli/bootstrap.go

```diff
@@ -296,6 +296,23 @@ $$;`, step.Artifact.URI, step.ID))
 		data.MongoDB.ImageArtifact.URI = artifactURIs["mongodb"]
 	}
 
+	// set TELEMETRY_CLUSTER_ID
+	telemetryClusterID := random.UUID()
+	sqlBuf.WriteString(fmt.Sprintf(`
+UPDATE releases SET env = jsonb_set(env, '{TELEMETRY_CLUSTER_ID}', '%q')
+WHERE release_id = (SELECT release_id FROM apps WHERE name = 'controller');
```

Comment:
Yes, the `TELEMETRY_CLUSTER_ID` is new for every restore, and the `TELEMETRY_BOOTSTRAP_ID` is generated once and stays the same across multiple restores.